doc/configuration.rst

```diff
@@ -598,6 +600,61 @@ examples into a single mini-gallery, e.g.:
 For such a mini-gallery, specifying a custom heading message is recommended
 because the default message is vague: "Examples of one of multiple objects".
 
+.. _file_based_minigalleries:
+
+Create mini-galleries using file paths
+======================================
+
+Sometimes you may want to explicitly create a :class:`mini-gallery <sphinx_gallery.directives.MiniGallery>` 
+for files that may be in the same location but not have functions in common, for example a set of 
+tutorials. The mini-gallery directive therefore supports passing in:
+
+* full qualified name of object
+* pathlike strings to example Python files
+* glob-style pathlike strings to example Python files
+
+.. code-block:: rst
+
+    .. minigallery:: numpy.exp ../examples/plot_0_sin.py ../examples/plot_4*
+
+
+.. minigallery:: numpy.exp ../examples/plot_0_sin.py ../examples/plot_4*
+    :add-heading: Mini-gallery using paths
+   :heading-level: ^
+
+Sort mini-gallery thumbnails from files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ordering of thumbnails can be set using the ``minigallery_sort_order`` 
```

Comment:
Can we make explicit that all minigalleries created with the `minigallery` directive will use the single `minigallery_sort_order` config value?

Reply:
Ohh sorry I was not clear and got mixed up. I did not mean to include backreferences. I mean to make it clear that the same `minigallery_sort_order` is used for all file minigalleries , but now that I think about it I don't think you'd ever want use different sort keys for different galleries. Resolving.